themis_components/thFilter/models/filterBase.service.spec.coffee

```
@@ -0,0 +1,28 @@
+describe "ThemisComponents: Service: FilterBase", ->
+  FilterBase = filterBase = null
+
+  beforeEach ->
+    angular.mock.module "ThemisComponents"
+
+  beforeEach inject ($injector, _FilterBase_) ->
+    FilterBase = _FilterBase_
+
+  describe "#constructor", ->
+    it "should return a new FilterBase object", ->
+      filterBase = new FilterBase {
+        fieldIdentifier: "id"
+        name: "name"
+      }
+      expect(filterBase.fieldIdentifier).toBe "id"
+      expect(filterBase.name).toBe "name"
+
+  describe "#getValue", ->
+    it "should throw an error", ->
```

Suggest "should throw an error when called directly" or something a little more explicit of what is being tested.